### ISSUE TITLE
Swap Whitelist <--> Blacklist?

### ISSUE THREAD

**Issue by Gavrilian** [ray-lothian/UserAgent-Switcher#46]
Rather small, and it might come down to taste or I'm just reading it wrong, but shouldn't Whitelist and Blacklist be swapped? So Blacklist would be specifying domains that you want to spoof, and Whitelist would specify domains that you want to exclude from spoofing. I figure its a pretty minor thing, but I thought it might be helpful for new users who don't read descriptions.

**Comment by Grossdm** [comment]
I agree that the terms `whitelist` and `blacklist` should be switched.


The concept of a whitelist is a set of items that will _not_ be acted upon.
Inversely, a blacklist is a set of items that _will_ be acted upon.

**Comment by Luctins** [comment]
from what I've tested it seems that the description text is inverted too.
